app/assets/locales/android_translatable_strings.txt

```diff
@@ -376,11 +376,22 @@ connection.task.success=No problems were detected.
 connection.task.report.commcare.popup=Your logs are being sent to CommCare!
 connection.task.unset.posturl=There has been an error on CommCare's end. Please report this problem to CommCare Support.
 
+connection.check.title=Checking Your Network Connection...
+connection.check.message=We are checking your network connection before proceeding with actual network call.

```

Comment:
"actual netowrk call" seems a bit techy for users. Can we make these strings specific to the process they are related to like syncing, app install etc ?

Reply:
==. It's fine to just say "We are checking the setup of your network connection" or something